tests/com/magento/idea/magento2plugin/reference/xml/MftfEntityNameReferenceRegistrarTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+package com.magento.idea.magento2plugin.reference.xml;
+
+public class MftfEntityNameReferenceRegistrarTest extends ReferenceXmlFixtureTestCase {
+
+    public void testCreateDataActionGroupMustHaveReference () {
```

Comment:
🤔 Is there any specific method declaration in Java?
```suggestion
    public void testCreateDataActionGroupMustHaveReference() {
```

Reply:
No, it's typo

Reply:
:)